rtengine/imagedata.h

```diff
@@ -21,36 +21,24 @@
 #include <cstdio>
 #include <memory>
 #include <string>
-#include <vector>
+#include <glibmm.h>
+#include <exiv2/exiv2.hpp>
 
-
-#include <libiptcdata/iptc-data.h>
-
-#include "imageio.h"
-
-namespace Glib
-{
-
-class ustring;
-
-}
-
-namespace rtexif
-{
-
-class TagDirectory;
-}
+#include "rawimage.h"
+#include "rtengine.h"
 
 namespace rtengine
 {
 
-class FrameData final
-{
-
-protected:
-    rtexif::TagDirectory* frameRootDir;
-    IptcData* iptc;
+Exiv2::Image::AutoPtr open_exiv2(const Glib::ustring &fname); // TODO: Global function?
 
+class FramesData :
+    public FramesMetaData
```

Comment:
```suggestion
class FramesData final :
    public FramesMetaData
```
https://github.com/Beep6581/RawTherapee/blob/33a4737c89d9b7450532908d5ef6477d47a5f08e/rtengine/imagedata.h#L104

Reply:
Done in #6631